modules/persistent-storage-csi-migration-automatic-ga.adoc

```diff
@@ -12,6 +12,8 @@
 
 * OpenStack Cinder
 
+* Google Compute Engine Persistent Disk (in-tree) and Google Cloud Platform Persistent Disk (CSI)
```

Comment:
for OSP and Azure we just mention OpenStack Cinder & Azure Disk why do we specify in-tree / CSI here?

Reply:
+1, I'd use `Google Compute Engine Persistent Disk (GCP PD)` because that's how they call it upstream: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver

Reply:
Done.